buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1982,17 +1982,25 @@ class BeamModulePlugin implements Plugin<Project> {
         return argList.join(' ')
       }
 
-      project.ext.toxTask = { name, tox_env ->
+      project.ext.toxTask = { name, tox_env, needsExpansionServiceJar = true ->
```

Comment:
wouldn't it be preferable to make false the default? I'd think most tox tasks do not need this.

Reply:
Yeah, definitely. I made it true as default without thinking at the moment of writing it.